app/services/foreman/plugin.rb

```diff
@@ -351,6 +351,11 @@ def parameter_filters(klass)
       @parameter_filters.fetch(klass.is_a?(Class) ? klass.name : klass, [])
     end
 
+    # Add concerns to a host
+    def extend_host(concern_module)
```

Comment:
I added it in response to [Marek's request](https://github.com/theforeman/foreman/pull/3701#issuecomment-238601625).